kingpin/actors/aws/elb.py

```diff
@@ -42,7 +43,7 @@
 EXECUTOR = futures.ThreadPoolExecutor(10)
 
 
-class ELBNotFound(exceptions.RecoverableActorFailure):
+class ELBNotFound(exceptions.UnrecoverableActorFailure):
```

Comment:
I'm conflicted as to how we should be using recoverable errors. Operating on a non existent ELB is 100% impossible and should always fail. Hence unrecoverable